### PR TITLE
Run iceqube tests on postgres.

### PR DESCRIPTION
**Pull request by rtibbles**
## Summary
* Supersedes https://github.com/learningequality/kolibri/pull/7295
* Ensures that iceqube tests run on postgres by using kolibri options to set postgres settings in tests rather than a separate settings file
* Adds client_encoding argument when connecting to postgres with SQLAlchemy

## Reviewer guidance
If tests still pass this should be good to go.

----

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [x] Critical and brittle code paths are covered by unit tests

## PR process

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: kolibri/core/utils/sqlalchemy.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+from django.db import connections
+from sqlalchemy.pool import NullPool
```
Comment (bjester) [inline kolibri/core/utils/sqlalchemy.py]: Would [`from __future__ import absolute_import`](https://github.com/learningequality/kolibri/blob/develop/kolibri/core/tasks/apps.py#L1) potentially cause this?
I notice the same error here: https://stackoverflow.com/a/33744089

Comment (bjester) [reply kolibri/core/utils/sqlalchemy.py]: Specifically because the file is the same name of the module

Comment (rtibbles) [reply kolibri/core/utils/sqlalchemy.py]: That's the ticket!

**Review by bjester — APPROVED** [review]
:100:

**Pull request merged**